package.json

```
@@ -17,7 +17,8 @@
 		"changeset:version": "changeset version && pnpm -r generate:version && git add --all",
 		"changeset:release": "changeset publish",
 		"start": "cd sites/kit.svelte.dev && npm run dev",
-		"postinstall": "pnpm -r generate:types"
+		"build": "pnpm --filter @sveltejs/* -r build",
+		"postinstall": "pnpm -r generate:types && pnpm build"
```

`pnpm install` will already run the `postinstall` on `kit`, which runs `generate:types`. it'd drive me crazy to have build the adapters everytime I install because I'm usually just working on `kit` and so it'd just be wasted time. I think we should have users run `build` themselves after `install`
```suggestion
		"build": "pnpm --filter @sveltejs/* -r build"
```

---

Having to remember to run build as well feels like higher work than the small additional time to have it run on postinstall IMO

---

I don't have to remember to run build though because it's not a thing that I ever need to do, so it's just time wasted sitting there with no benefit